src/resources/team/team.router.ts

```diff
@@ -1,15 +1,34 @@
 import {Router} from 'express'
 import {Permission, protect} from '../../middlewares/permission'
 
-import {validateCreateTeam} from './team.validator'
+import {validateCreateTeam, validateGetTeams} from './team.validator'
 import * as teamController from './team.controller'
 
 const router = Router()
 
 const writeTeam = protect([Permission.WriteTeam])
+const readTeam = protect([Permission.ReadTeam])
 
 // POST request to create a new team
 router
+	.route('/')
+	/**
```

Comment:
I suggest the endpoint for querying teams for one user is GET /users/: id/teams. Consider using GET /users/me/teams since we don't have use case to query teams of other users. So put this endpoint to user route instead